csharp/autobuilder/Semmle.Autobuild.CSharp/Constants.cs

```diff
@@ -1,8 +0,0 @@
-namespace Semmle.Autobuild.CSharp
-{
-    internal static class Constants
-    {
-        // The version number should be kept in sync with the version .NET version used for building the application.
-        public const string LatestDotNetSdkVersion = "8.0.101";
```

Comment:
nit: Could you also make a PR in the csharp team repo and update the guide for updating .NET (now this doesn't need to updated anymore).